go/vt/sqlparser/sql.y

```diff
@@ -302,7 +303,7 @@ func bindVariable(yylex yyLexer, bvar string) {
 
 // DDL Tokens
 %token <str> CREATE ALTER DROP RENAME ANALYZE ADD FLUSH CHANGE MODIFY DEALLOCATE
-%token <str> REVERT
+%token <str> REVERT QUERIES EXHAUSTIVE
```

Comment:
`EXHAUSTIVE` is no longer needed, I think

Reply:
Done